core/modules/user/user.module

```diff
@@ -710,6 +715,7 @@ function user_account_form(&$form, &$form_state) {
   $register = ($form['#user']->uid > 0 ? FALSE : TRUE);
 
   $admin = user_access('administer users');
```

Comment:
Perhaps rename this variable to `$admin_users`, so that there's more consistency in the code where these variables are being used.

Reply:
Good idea! Done, and PR updated.